tests/integration/pdo/mysql/constructor/test_instance_reporting_2.php

```diff
@@ -0,0 +1,161 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*DESCRIPTION
```

Comment:
This whole test seems duplicate of the same named [one in the base-class directory](https://github.com/newrelic/newrelic-php-agent/pull/993/files#diff-a28747bcacf6e28f41e0a8e6b95933d73810cfabc008fde280cf55ed9d815b31R8).

Shouldn't it be the `PDO's specialized subclass constructor is used to create connection object` version?